pipeline_tools/tests/data/analysis_protocol.json

```diff
@@ -0,0 +1,11 @@
+{
+  "computational_method": "SmartSeq2SingleCell",
+  "describedBy": "http://schema.integration.data.humancellatlas.org/type/protocol/analysis/7.1.0/analysis_protocol",
```

Comment:
We're using analysis_protocol version 8.0.0 now right? Does that require updating this test data as well?

Reply:
Good catch, I will update the testing data shortly, thanks!